src/server/native_controllers/native_controller_start.js

```diff
@@ -0,0 +1,36 @@
+require('../../babel-register');
+const program = require('commander');
+
+program
+  .arguments('<nativeControllerPath> [config...]')
```

Comment:
IMO, `config` should use a standard encoding format, rather than a list of `=` separated key-value pairs. Why not just take `configJSON` as a second argument so that we don't have to worry about what happens when someone sends a key or value with `=`, spaces, etc.